agent/internal/util/json_test.go

```diff
@@ -18,8 +18,8 @@ func TestRemoveJSONComment(t *testing.T) {
 			  "i":"o",//
 "kek": "// //"//
 }`
-	expectstr := `{				            	    	  			  "json" : true, 			  "i":"o","kek": "// //"}`
+	expectstr := `{			  "json" : true,			  "i":"o","kek": "// //"}`
```

Comment:
"example" or "test" would be a better text instead of "kek" :D

Reply:
I did not write this test, just adjusted it after improving the regex, `kek` was already there.

Reply:
Oh okay, nvm then :)